Felienne/hedy
# [BUG FIX] Explore UI stays consistent independent of username or program length

**Description**
In this PR we fix an issue where due to a too long username or long program the layout would shift within the explore program box. A typo is corrected and the styling is improved so this issue can no longer occur. Also an incorrect created `staticr/css` folder is removed.

**Fix for**
This PR fixes #1803

**How to test**
Visit the explore page and make sure several programs exists with usernames longer than 15 or 20 characters and programs which required horizontal scrolling (long lines). Notice that the UI stays consistent.
